api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -166,6 +166,7 @@ message ClusterManager {
   // <envoy_api_field_core.ApiConfigSource.api_type>` :ref:`GRPC
   // <envoy_api_enum_value_core.ApiConfigSource.ApiType.GRPC>`.
   envoy.api.v2.core.ApiConfigSource load_stats_config = 4;
+  envoy.api.v2.core.ApiConfigSource hds_config = 5;
```

Comment:
Can you add a comment here explaining what HDS is? Ideally add a `:ref:` link back to the proto definition as in the above example for the load stats reporter.